dirs/dirs.go

```diff
@@ -154,6 +155,7 @@ func SetRootDir(rootdir string) {
 	SnapAssertsSpoolDir = filepath.Join(rootdir, "run/snapd/auto-import")
 
 	SnapStateFile = filepath.Join(rootdir, snappyDir, "state.json")
+	SnapSystemKeyFile = filepath.Join(rootdir, snappyDir, "system-key")
```

Comment:
The name "key" implies this is somehow related to crypto. How about `system.yaml`, with less indirection?